core/storage/base_model/gae_models.py

```diff
@@ -891,15 +909,27 @@ def get_snapshot_id(cls, instance_id: str, version_number: int) -> str:
         return '%s%s%s' % (
             instance_id, VERSION_DELIMITER, version_number)
 
+    def _prepare_additional_models(self) -> Mapping[str, BaseModel]:
+        """Prepares additional models needed for the commit process. The dict
+        is empty by default  and this method should be overriden in models that
```

Comment:
Extra space before "and".

When you say "the dict", I am not sure what you are referring to here.

Reply:
Rewritten.